calico.go

```diff
@@ -92,6 +92,10 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
+	if ready := IsReady(calicoClient); !ready {
+		return fmt.Errorf("Calico is currently not ready to process requests")
```

Comment:
What do you think about adding a line saying, upgrade might be in progress, so users don't wonder why it's not ready

Reply:
OR maybe a warning log before the return line saying "upgrade might be in progress" or something